### ISSUE TITLE
Feature request: Display time until code invalidation

### ISSUE THREAD

**Issue by maddsua** [yitsushi/totp-cli#88]
I think it should be reasonably easy to print time in seconds that a code will still be valid, something that Google Authenticator displays with that pie icon.

Some providers will allow for a few seconds of 'delay', but the other would outright reject a code.

So the output would be something like:
```bash
totp-cli generate namespace account
Password: ***
889840 | 25s
```

**Comment by yitsushi** [comment]
Available in [v1.7.0](https://github.com/yitsushi/totp-cli/releases/tag/v1.7.0).

```
❯ go run . generate --show-remaining namespace account
Password: ***
465669 (remaining time: 26s)
```

**Comment by maddsua** [comment]
> Available in [v1.7.0](https://github.com/yitsushi/totp-cli/releases/tag/v1.7.0).
> 
> ```
> ❯ go run . generate --show-remaining namespace account
> Password: ***
> 465669 (remaining time: 26s)
> ```

Great! Didn't see it in the embedded help command and so I've assumed that it just isn't there

**Comment by yitsushi** [comment]
It wasn't there, it's there only since v1.7.0, which contains only the #89 pull request.
